sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/CopyableFileSmbProperties.java

```diff
@@ -10,12 +10,12 @@
  * SMB Properties to copy from the source file.
  */
 public enum CopyableFileSmbProperties {
-    NONE("none"),
-    FILE_ATTRIBUTES("fileattributes"),
-    CREATED_ON("createdon"),
-    LAST_WRITTEN_ON("lastwrittenon"),
-    CHANGED_ON("changedon"),
-    ALL("all");
+    IS_NONE("none"),
```

Comment:
Do most enums in the Java Storage SDK start with `IS_`?  My comment in the API view around the `is` prefex is limited in scope to boolean get() method.